client/src/components/Brand.tsx

```diff
@@ -1,68 +1,43 @@
 import { Link } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { readBrands } from '../library/data';
+
+type Brands = {
+  brand: string;
+};
 
 type BrandProps = {
   isOpen: boolean;
   onClose: () => void;
 };
 
 export function Brand({ isOpen, onClose }: BrandProps) {
+  const [brands, setBrands] = useState<Brands[]>([]);
+
+  useEffect(() => {
+    async function load() {
+      try {
+        const brands = await readBrands();
+        setBrands(brands);
+        console.log(brands);
+      } catch (error) {
+        console.error(error);
```

Comment:
If all you do is log an error, the user gets no indication that an error happened. This can be confusing because they won't realize something went wrong and might even believe things worked correctly when they didn't.

You should always display errors to the user. Just an `alert` is OK, though most apps these days display errors in a toast bar or something similar.